images/dockerregistry/Dockerfile.centos7

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
is there a reason we have this file? it's identical to Dockerfile.

Reply:
> is there a reason we have this file? it's identical to Dockerfile.

I have no idea.  I assumed that CI might want it and it didn't cost me anything to update.

Reply:
No need for this for CI

Reply:
so does anyone know why we have it?  (we seem to have it for other images too).